_rules/link-in-context-descriptive-5effbb.md

```diff
@@ -104,6 +104,16 @@ The [accessible name][] describes the purpose of the link.
 
 #### Passed Example 3
 
+The [accessible name][] together with the [accessible description][] describes the purpose of the link.
```

Comment:
I feel we should also add an example with `aria-describedby`.

Reply:
added example